pkg/auth/auth.go

```diff
@@ -368,33 +367,33 @@ func (a *Authenticator) CallbackFunc(fn func(loginInfo LoginJSON, successURL str
 		}
 
 		if code == "" {
-			log.Infof("missing auth code in query param")
+			klog.V(4).Info("missing auth code in query param")
```

Comment:
Let's drop the verbosity here, since previously we havent been using `log.Debug()` here.